internal/builders/generic/README.md

```diff
@@ -654,7 +654,7 @@ If you use [Haskell](https://www.haskell.org/) (either via
 [`cabal`](https://www.haskell.org/cabal/) or
 [`stack`](https://docs.haskellstack.org/en/stable/README/)) to generate your
 artifacts, you can easily generate SLSA3 provenance by updating your existing
-workflow with the 5 steps indicated in the workflow below.
+workflow with the 3 steps indicated in the workflow below.
```

Comment:
just remove the number of steps, this way we won't screw it up. `workflow with the steps`

Reply:
Ah, you did it before me :)